### ISSUE TITLE
tiptap-floating-menu is visible at all time

### ISSUE THREAD

**Issue by dkrusenstrahle** [sibiraj-s/ngx-tiptap#3]
Hello,

The tiptap-floating-menu component is visible at all time and does not appear within the editor (no matter what I put inside of the component). The tiptap-bubble-menu works fine with the same content. I even tried the code from the demo project.

This is how I use it:

```
<div class="component-editor editor-bordered">
  <tiptap [editor]="editor"></tiptap>
</div>

<tiptap-bubble-menu [editor]="editor">
  <div class="component-editor-toolbar-inline">
    <div class="toolbar-button" (click)="editor?.chain().focus().toggleHeading({ level: 1 }).run()"><img src="assets/images/editor/h1.svg"></div>
    <div class="toolbar-button" (click)="editor?.chain().focus().toggleHeading({ level: 2 }).run()"><img src="assets/images/editor/h2.svg"></div>
    <div class="toolbar-button" [ngClass]="{active: editor?.isActive('bold')}" (click)="editor.chain().focus().toggleBold().run()"><img src="assets/images/editor/text--bold.svg"></div>
    <div class="toolbar-button" [ngClass]="{active: editor?.isActive('italic')}" (click)="editor.chain().focus().toggleItalic().run()"><img src="assets/images/editor/text--italic.svg"></div>
    <div class="toolbar-button" (click)="editor?.chain().focus().setTextAlign('left').run()"><img src="assets/images/editor/text--align--left.svg"></div>
    <div class="toolbar-button" (click)="editor?.chain().focus().setTextAlign('right').run()"><img src="assets/images/editor/text--align--right.svg"></div>
  </div>
</tiptap-bubble-menu>

<tiptap-floating-menu [editor]="editor">
  <div class="component-editor-toolbar-floating">
    <div class="toolbar-button" (click)="editor?.chain().focus().toggleHeading({ level: 1 }).run()"><img src="assets/images/editor/h1.svg"></div>
    <div class="toolbar-button" (click)="editor?.chain().focus().toggleHeading({ level: 2 }).run()"><img src="assets/images/editor/h2.svg"></div>
    <div class="toolbar-button" (click)="editor?.chain().focus().insertTable({ rows: 3, cols: 3, withHeaderRow: true }).run()"><img src="assets/images/editor/table.svg"></div>
    <div class="toolbar-button" (click)="openPopup()"><img src="assets/images/editor/image.svg"></div>
    <div class="toolbar-button" (click)="editor?.chain().focus().toggleBulletList().run()"><img src="assets/images/editor/list.svg"></div>
  </div>
</tiptap-floating-menu>
```

**Comment by sibiraj-s** [comment]
Yeah Even I had issues with floating menu component while testing as well. Tests are not added yet. 
But it works locally though.

Will look into it.

**Comment by dkrusenstrahle** [comment]
Strange. I am testing it locally too if that helps.

**Comment by sibiraj-s** [comment]
You can clone this repo and check whats different. See Contributing.md for how to get started. https://github.com/sibiraj-s/ngx-tiptap/blob/master/.github/CONTRIBUTING.md.

I am occupied at personal work here. I will check this tonight.

**Comment by sibiraj-s** [comment]
were you able to fix this? can you share a small reproducible repo reproducing this?

**Comment by dkrusenstrahle** [comment]
Unfortunately not

**Comment by sibiraj-s** [comment]
Thanks for finding it. Fixed in v1.2.1.
